### ISSUE TITLE
destination host unreachable

### ISSUE THREAD

**Issue by Luna-maria90** [tj0105/source_routing#2]
I'm trying the pof version of the source routing algorithm, I run the test and start the pof controller,
however when I try to ping from h1 to h2 the ping is failed.
Am I do something wrong?

**Comment by tj0105** [comment]
What you said is so simple that I don't know what specific problem you have. You can add me on QQ:1416880225

**Comment by Luna-maria90** [comment]
it is so simple these steps are what I did to run the source routing code :
1- cd source_routing/POF/mininet-pof/util
2- ./install.sh -nf
3- cd source_routing/POF
4- sudo python source_route_test1_test2.py 192.168.0.107
5- on another machine I run [sudo python ./pox.py sr_ldy  openflow.pof_01 --port=6666 web pofdesk_v5]
6- then from the mininet prompt I run the command [h1 ping h2] but the result was [destination host unreachable] 
This is what I did to, is there something missing? I'm new to pof switch, so can you pls. guid me.

**Comment by tj0105** [comment]
You determine whether the H1 force exists in the MAC address of the H2 device

**Comment by Luna-maria90** [comment]
sorry I don't understand what you mean by that, how I can do this?

edit:
if you mean the arp table of h2 then it has nothing in it , but the arp table of h1 is:
Address   -    10.0.0.2         
HWtype    
HWaddress  - (incomplete) 
Flags Mask           
 Iface-             h1-eth0

**Comment by tj0105** [comment]
You need to manually configure the arp table for h2 in h1, using "arp -s IP MAC  ".For h2, add the arp table for h1 as well

**Comment by Luna-maria90** [comment]
I do that but the result is the same [
![Screenshot ](https://user-images.githubusercontent.com/78085053/118072484-8f05ba80-b3b2-11eb-8f7c-5895993f2c14.png)
](url)

**Comment by tj0105** [comment]
Do you have POFSwitch installed?This is the file POF/POFSwitch_Groups, and you must install the POFSwitch before starting the topology.Installation steps are as follows:
1. cd POF/POFSwitch_Groups
2. sudo ./configure
3. sudo make
4. sudo make install

**Comment by Luna-maria90** [comment]
yes I had it installed, and I also reinstalled it with mininet, but nothing change.
